### DISCUSSION TITLE
Emoji and Custom icon ratings

### DISCUSSION THREAD

**Discussion by B3Kay** [saadeghi/daisyui#1653] [Ideas / feature requests]
It would be awesome with a Emoji or custom rating like this:
https://dribbble.com/shots/17097931/attachments/12190143?mode=media
![image](https://user-images.githubusercontent.com/23380102/222474647-3a5d9f95-a230-421f-84a5-75cebea3f618.png)

**Comment by saadeghi** [comment]
That doesn't need to be a daisyUI rating. because it's basically picking 1 option and only the picked option should be highlighted.  
What daisyUI's rating does is it highlights the chosen option and all previous options. like a giving stars in a star rating.

If you want these emojis, you can simply use 5 labels and put an emoji + a hidden radio button inside each label.

But if you really want to put emojis inside a daisyUI's star rating style, it can be done like this: https://play.tailwindcss.com/IWnA596xtw?file=css
